Sources/SwiftUIReactor/Reactor.swift

```diff
@@ -66,3 +66,51 @@ public extension Reactor {
         )
     }
 }
+
+/// Property wrapper to get a binding to a state keyPath and a associated Action
+@propertyWrapper
+public struct Action<R: Reactor, Value>: DynamicProperty {
+    @EnvironmentObject
+    var reactor: R
+    
+    let keyPath: KeyPath<R.State, Value>
+    let action: (Value) -> R.Action
+    
+    public init(_ reactorType: R.Type, keyPath: KeyPath<R.State, Value>, action: @escaping (Value) -> R.Action) {
```

Comment:
Should wochenende hide the action parameter name like in the binding functions?
Or is it then hard to understand and not readable?
If wochenende change it, it should be the same in both.

Reply:
Not sure how to do it. We could hide all the parameter names I think. Then it would be less to write.